### ISSUE TITLE
[Test Failure][Localization] The tooltips of search filter in PM UI are not localized

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#7716]
## Details about Problem
NuGet version: NuGet Client Release-5.0.0-Preview2\5.0.0.5782
VS Version: D16.0\28519.51
OS Version: 16299.RS3_Release_svc_escrow.180502-1908 CHS
## Detailed repro steps so we can see the same problem
1. Create a solution with any project in VS and open the PM UI of the project/solution.
2. Hover on the search filter/search button/search Dropdown button in the PM UI and check the tooltips.
## Expected
All the tooltips of the search filter (“Type words to search for”/”Search”/”Show Dropdown”) should be localized correctly.
## Actual
The tooltips of search filter (“Type words to search for”/”Search”/”Show Dropdown”) in PM UI are not localized as below screenshot.
![tooltipnotlocalized](https://user-images.githubusercontent.com/20632625/51528253-34f32b00-1e71-11e9-92f6-f03344c058c8.png)

## Note
1.It repro on all languages for D16.0\28519.51.
2.It didn’t repro on D16.0 28327.66 with Default NuGet Client Dev\5.0.0.5663.

**Comment by dominoFire** [comment]
Confirmed. Those were correctly located in d15.0

**Comment by johnliu369** [comment]
Assume this bug will be gone once VS client translation recovered.
